Mathlib/Tactic/SolveByElim.lean

```diff
@@ -117,20 +116,36 @@ By default, the assumptions passed to `apply` are the local context, `rfl`, `tri
 `congrFun` and `congrArg`.
 
 The assumptions can be modified with similar syntax as for `simp`:
-* `solve_by_elim [h₁, h₂, ..., hᵣ]` also applies the named lemmas.
-* (not implemented yet) `solve_by_elim with attr₁ ... attrᵣ` also applies all lemmas tagged with
-  the specified attributes.
+* `solve_by_elim [h₁, h₂, ..., hᵣ, attr₁, ... attrᵣ]` also applies the named lemmas, as well as
+  all lemmas tagged with the specified attributes.
```

Comment:
@digama0 has support been added to `solve_by_elim` so that it now with with attributes like this? If not, this comment should be marked "not implemented yet".